### ISSUE TITLE
Working on aereg (libWIN32Access) Using Visual Studio

### ISSUE THREAD

**Issue by soroosh-chabi** [BiglySoftware/BiglyBT#2922]
While working on #1870 I needed to examine the native code in libWIN32Access. Apparently it is a Visual Studio solution with only one project. The project seems to have been created with a really old version of Visual Studio (likely Visual Studio 6 as evidenced by .dsp and .dsw files) and later updated to Visual Studio 2010 (as evidenced by the contents of .vcsproj file). When importing it in Visual Studio 2022 I had to upgrade the Platform Toolset and Windows SDK version used to build the project and even then there were other compilation errors that needed to be dealt with. I also examined the Git log and saw no updates to this directory ever since the initial commit.

Is there any other method of working with this part of the code base or if, as the Git log suggests, no one is actually working with it, are there any plans or guidelines for bringing it to such a state?

**Comment by parg** [comment]
As you say, nobody's touched this piece of crap forever... Obviously it would be good to get it into a compilable state but it'd have to still work with Windows XP at least as there are some users on that...

Personally it isn't something I have any interest in working on until something goes wrong with it!

**Comment by soroosh-chabi** [comment]
When getting aereg to compile in Visual Studio 2022 whilst targeting Windows XP and above, again I had to overcome several obstacles since Windows XP is completely out of support and Visual Studio has deprecated the toolset that can target it so it is hard to find good documentation on that.

When time came to test the build results on Windows XP, I setup a Virtual Box VM with Windows XP SP3 but couldn't get the latest version of BiglyBT to install as it failed while downloading the JRE. I downloaded the JRE the installer was trying to download but still got errors when I got the installer to use it or when I tried running its `java` executable . The MSI installer for the latest Java 1.8.0 release from Adoptium could not be installed. The Zip variant couldn't start. Had a similar experience with Windows XP x64 SP2. But of course, [this was all to be expected](https://adoptium.net/supported-platforms/).

As a result, I'm wondering is supporting Windows XP really worth all of this. After all, even under the BiglyBT Windows installer download buttons it says:
>WinXP works, but **unsupported**
